torntools/scripts/js/globalFunctions.js

```diff
@@ -1507,10 +1507,10 @@ const STORAGE = {
 				highlight_bloodbags: "none",
 				show_missing_plushies: false,
 				show_missing_flowers: false,
-				show_book_effects: true,
-				show_alcohol_nerve_gains: true,
-				show_candy_happy_gains: true,
-				show_e_can_gains: true,
+				show_book_effects: true, // Debate: false by default
+				show_alcohol_nerve_gains: true, // Debate: false by default
+				show_candy_happy_gains: true, // Debate: false by default
+				show_e_can_gains: true, // Debate: false by default
```

Comment:
These 3 don't mess with the UI at all so I think these can be left on by default.

Reply:
Yeah, I personally disabled them, but I do think that these should be left on by default. Someone asked for them to be off by default, which is why I added it here.

Reply:
But they can be personally disabled just like you did and these are helpful for a lot of ppl(mainly noobs) who still don't know that fac and job perks increase nerve and happy gains. Also, it can save a lot of time for people who calculate the gains themselves.

Reply:
I think these can be on by default, don't think many people want to turn them off anyway.

But I just checked all 3 of them and they should be unified a bit. Example screenshots:
![image](https://user-images.githubusercontent.com/20304303/106645781-7dcc3700-6595-11eb-87bf-ae520c8b93bc.png)
![image](https://user-images.githubusercontent.com/20304303/106645798-815fbe00-6595-11eb-809a-93887cb30384.png)
![image](https://user-images.githubusercontent.com/20304303/106645807-84f34500-6595-11eb-9fdf-6f486b69d83f.png)

I personally would change the format to normal text and write the stat name out e.g. "30energy", "1nerve", "25happy".